src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -120,4 +125,11 @@ export default class ColorPicker extends React.PureComponent<Props, State> {
 
     onChange({hue, saturation, brightness, alpha});
   };
+
+  private handlePickerDrag = (
+    evnt: React.MouseEvent<HTMLDivElement, MouseEvent>,
```

Comment:
For code consistency:

```suggestion
    event: React.MouseEvent<HTMLDivElement, MouseEvent>,
```